setup.py

```diff
@@ -54,7 +54,7 @@
         "jieba==0.42.1",
         "comment_parser",
         "beautifulsoup4",
-        "mwparserfromhell @ git+https://github.com/earwig/mwparserfromhell.git@0f89f44",
+        "mwparserfromhell==0.6.5",
```

Comment:
I have verified that this version has no differences in output from the commit that we previously pinned at.